source/launch.txt

```diff
@@ -24,13 +24,16 @@ databases. You have several options for creating a schema and
 launching :binary:`~bin.mongosqld`. This guide will help you choose
 the best option for your needs.
 
-By default, :binary:`~bin.mongosqld` generates a :doc:`data schema
-</schema-configuration>` and holds it in memory. Alternatively, if
-you prefer to create a schema file and edit it manually, use the
-:binary:`~bin.mongodrdl` program to create a ``.drdl`` schema
-file and use the :option:`--schema <mongosqld --schema>` option
-when starting :binary:`~bin.mongosqld`. For more information on schema
-generation and data sampling, see :doc:`schema-configuration`.
+By default, :binary:`~bin.mongosqld` generates a default :doc:`data schema
+</schema-configuration>` and stores it in memory. To create a custom schema
+instead and edit it manually, perform both of the following actions:
+
+- Use the :binary:`~bin.mongodrdl` program to create a custom ``.drdl`` schema
+  file and provide a custom name for this file using the :commandoption:`name-schema <name-schema>` command.
+- Start :binary:`~bin.mongosqld` with the :option:`--schema <mongosqld --schema>` and the
```

Comment:
Replace the new line 33 and 34 with the following:
Start the mongosqld with the `--schemaSource` and `--schemaName` option if the schema is previously stored in the database with [mongodrdl upload](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/docsworker-xlarge/DOCSP-29384/reference/mongodrdl/#mongodb-commandoption-commandoption.upload).